packages/contact-center/cc-components/src/components/task/task.types.ts

```diff
@@ -109,7 +109,7 @@ export interface TaskProps {
   /**
    * List of tasks
    */
-  taskList: Record<string, ITask>;
+  taskList: Record<string, ITask> | null | undefined;
```

Comment:
```suggestion
  taskList: Record<string, ITask> 
```
To make it optional use <Partial <Pick TaskProps, 'taskList' | 'currentTask'>>

See ,
<img width="746" height="183" alt="Screenshot 2025-07-22 at 10 44 44 AM" src="https://github.com/user-attachments/assets/fc6c672f-8f88-4fa1-a760-a58559b8ed98" />